### DISCUSSION TITLE
Channel programming with chirp doesn't work correctly

### DISCUSSION THREAD

**Discussion by mentanah** [egzumer/uv-k5-firmware-custom#280] [General]
When using the .20x firmware with the egzumer [chirp module](https://github.com/egzumer/uvk5-chirp-driver) and the latest chirp software to program channels, 
settings like AM modulation or Tx0Dir are programmed but not taking effect in device. 
Even they are read back from radio correctly afterwards.

**Comment by egzumer** [comment]
Readme updated: https://github.com/egzumer/uvk5-chirp-driver?tab=readme-ov-file#introduction

**Reply by mentanah** [reply]
I got it!
